src/components/ChatApp/MessageListItem/generateMessageBubbleUtil.js

```diff
@@ -92,7 +92,12 @@ const generateGifBubble = (
     <MessageContainer key={action + index}>
       <MessageBubble author={message.authorName}>
         <div>
-          <iframe src={gifSource} frameBorder="0" allowFullScreen />
+          <iframe
+            src={gifSource}
+            title="gifFrame"
```

Comment:
Here the title need not be camelCase. Write proper title, like `SUSI features GIF`

Reply:
@samagragupta This change was not made.